### ISSUE TITLE
crhc-linux-x64 binary won't work if /tmp is mounted with noexec option

### ISSUE THREAD

**Issue by sayan3296** [C-RH-C/crhc-cli#183]
The crhc script or the crhc-linux-x64 binary itself will not work and throw the following error if /tmp is mounted with noexec option in that system:

```
# ./crhc-linux-x64
./crhc-linux-x64: error while loading shared libraries: libz.so.1: failed to map segment from shared object
```

https://github.com/C-RH-C/crhc-cli/blob/master/conf/conf.py#L8-L17 forces\defaults the directory to be  /tmp. 

Hence. Either the error handling needs to be improved here or an option should be introduced to pass a different tmpdir for crhc .

**Comment by sayan3296** [comment]
CC @waldirio

**Comment by waldirio** [comment]
Hello @sayan3296 

Thank you for the heads up and sorry for my delay here.

I believe we can add a flag that would change this directory, if necessary, and by default, it will be using the standard. What do you think?

Meanwhile, I'll add a note on the kcs/article.

Thank you again!
Waldirio

**Comment by sayan3296** [comment]
Yeah.. A flag sounds perfect.. As long as user can specify a directory when default one cannot be used, we should be good.

**Comment by waldirio** [comment]
@sayan3296 

Perfect. Article update and I'll be working on this flag soon.

If you would like to move on here and push the PR, it will be very welcome!

teamwork++ and thank you in advance!
Waldirio
